src/main/java/com/example/User.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public interface User {
```

Comment:
`User` should not be an interface, typically is a class with fields `username, password, ...` and the getter methods